man/kernel-command-line.xml

```diff
@@ -85,6 +85,7 @@
         <term><varname>systemd.crash_shell</varname></term>
         <term><varname>systemd.crash_reboot</varname></term>
         <term><varname>systemd.confirm_spawn</varname></term>
+        <term><varname>systemd.disable_watchdog</varname></term>
```

Comment:
not a fan of "negative" options, i.e. "disable_foo=bool", where "yes" means "off" and "no" means "on"...

moreover, I think we should keep this closer to the existing options in system.conf, i.e. expose this as:

systemd.runtime_watchdog_sec= and systemd.shutdown_watchdog_sec= or so, and take the same arguments as RuntimeWatchdogSec= and ShutdownWatchdogSec=, with 0 or "infinity" meaning "off".